### PR TITLE
refactor settings controller to use programmatic UI

### PR DESCRIPTION
**Pull request by kingsqd**
In this PR I am refactoring settings to be ready to use programmatic UI.

Simulator Screen Shot - iPhone 11 - 2020-06-04 at 01 51 51

### CODE REVIEW

**Comment by kingsqd** [comment]
@Sharkesm good catch ! just updated

File: Say Their Names/Source/Components/MainTabBarController.swift [hunk 1]
```diff
@@ -95,12 +95,12 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
...
         petitionsNC.tabBarItem.title = "Petitions"
         
-        settingsNC.isNavigationBarHidden = true
-        settingsNC.tabBarItem.image = #imageLiteral(resourceName: "settings")
-        settingsNC.tabBarItem.selectedImage = UIImage(named: "settings_active")
-        settingsNC.tabBarItem.title = "Settings"
+        moreNC.isNavigationBarHidden = true
+        moreNC.tabBarItem.image = #imageLiteral(resourceName: "settings")
+        moreNC.tabBarItem.selectedImage = UIImage(named: "settings_active")
+        moreNC.tabBarItem.title = "More"
```
Comment (hybridcattt) [inline Say Their Names/Source/Components/MainTabBarController.swift:101]: now this uses `Strings.` (and there's a conflict)

File: Say Their Names/Source/Controller/Settings/MoreController.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//  SettingsController.swift
+//  Say Their Names
+//
+//  Created by Manase on 03/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class SettingsController: BaseViewController {
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Controller/Settings/MoreController.swift:11]: Duplicate. Please remove

File: Say Their Names/Source/Controller/Settings/MoreView.swift [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+//  Say Their Names
+//
+//  Created by Hakeem King on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Settings
+final class SettingsView: UIView {
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Controller/Settings/MoreView.swift:12]: Duplicate. Please remove

**Comment by kingsqd** [comment]
Just getting off my day job, will make the changes and push

File: Say Their Names/Source/Controller/More/MoreView.swift [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    private func setupSubviews() {
+        setupMoreLabel()
+    }
+}
+
+// MARK: - Configurations
+
+private extension MoreView {
+    
+    func setupMoreLabel() {
```
Comment (manish-d) [inline Say Their Names/Source/Controller/More/MoreView.swift:48]:
```suggestion
    private func setupMoreLabel() {
```

**Comment by kingsqd** [comment]
Closing this PR due to conflicts I have another one up for this

**Pull request closed without merging**